src/Integration.UnitTests/MefServices/ProjectPropertyManagerTests.cs

```diff
@@ -21,59 +21,27 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Windows.Threading;
 using EnvDTE;
 using FluentAssertions;
-using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
 using SonarLint.VisualStudio.TestInfrastructure;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests
 {
     [TestClass]
     public class ProjectPropertyManagerTests
     {
-        #region Test boilerplate
-
         private const string TestPropertyName = "MyTestProperty";
 
-        private ConfigurableVsProjectSystemHelper projectSystem;
-        private IHost host;
-
-        [TestInitialize]
-        public void TestInitialize()
```

Comment:
Most of this setup is no longer required, so I simplified the tests to remove `TestInitialize` and following our current testing conventions.